doc/install.rst

```diff
@@ -96,15 +98,13 @@ Installing PyGMT
 ----------------
 
 Now that you have GMT installed and your conda environment activated,
-use ``pip`` to install the latest source of PyGMT from Github::
+use ``pip`` to install the latest release of PyGMT from `PyPI <https://pypi.org/project/pygmt>`__::
 
-    pip install https://github.com/GenericMappingTools/pygmt/archive/master.zip
+    pip install pygmt
 
-Alternatively, you can clone the git repository and install using ``pip``::
+Alternatively, you can install the development version from the Github repository::
 
-    git clone https://github.com/GenericMappingTools/pygmt.git
-    cd pygmt
-    pip install .
+    pip install https://github.com/GenericMappingTools/pygmt/archive/master.zip
 
 This will allow you to use the ``pygmt`` library from Python.
```

Comment:
I guess we can leave the conda-forge instructions for later since I'm not entirely confident that it will work easily. So we can:

1.  release 0.1.0 without conda instructions
2. build the  conda-forge feedstock and make sure everything works
3. release 0.1.1 with only the docs changes or some minor fixes that may be required

This shouldn't be much work since most of the release is automated anyway.

Reply:
Was thinking along the same lines too. [Murphy's Law](https://en.wikipedia.org/wiki/Murphy's_law) might strike and I think we should leave it to the 0.1.1 release.